src/ashley/machina_extensions/forum_permission/defaults.py

```diff
@@ -0,0 +1,24 @@
+"""
+This module defines the default permissions that can be configured for a forum applications.
+"""
+
+from django.utils.translation import gettext_lazy as _
+from machina.apps.forum_permission.defaults import (
+    PermissionConfig as BasePermissionConfig,
+)
+
+
+class PermissionConfig(BasePermissionConfig):
+    """
+    Overrides the PermissionConfig class from machina forum_permission
+    application to add custom permissions.
```

Comment:
Module docstring were copied from the base machina application.
I thought they were satisfying enough, since the goal of each of these module hasn't changed.

I was more specific in the class that I overrided or added, to explain why I did that.